.github/workflows/docker.yml

```diff
@@ -1,9 +1,10 @@
 name: Docker Image
 on:
-  create:
```

Comment:
Let me know if you still want this @jsign. `create` is triggered any time someone creates a branch or tag. Seems like the most important tags are releases, and those are now covered by `release:` so I removed this.

Reply:
LGTM.